### DISCUSSION TITLE
Add a comment box on every single page

### DISCUSSION THREAD

**Discussion by alvoo123** [facebook/docusaurus#8460] [General]
Hi there,

I would like to add a comment box for every use to feedback, the administrator will have the means to moderate the comments and other users can view the comments as well.
What is the best way I can achieve this?

2ndly, I am new to this, what can I do to edit this?

**Comment by slorber** [comment]
Docusaurus does not provide a backend service to host comments and don't plan to. You'll have to use a service for that.

You can add whatever you want at the bottom of a doc or a blog post thanks to swizzling

https://dev.to/m19v/how-to-add-giscus-comments-to-docusaurus-439h

https://madrus4u.com/blog/blog-comments
